tools/src/main/scala/com/nvidia/spark/rapids/tool/ThreadFactoryBuilder.scala

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
May be we can change to use add-source to avoid copy it?

```
diff --git a/tools/pom.xml b/tools/pom.xml
index 1fb24656b..e5331324b 100644
--- a/tools/pom.xml
+++ b/tools/pom.xml
@@ -132,6 +132,24 @@
                 <groupId>net.alchim31.maven</groupId>
                 <artifactId>scala-maven-plugin</artifactId>
             </plugin>
+           <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-source</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                   <source>${project.basedir}/../common/src/main/scala/com/nvidia/spark/rapids/</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
```